engine/lib/input.php

```diff
@@ -538,6 +538,27 @@ function _elgg_htmlawed_test($hook, $type, $value, $params) {
 }
 
 /**
+ * Disable the autocomplete feature on password fields
+ *
+ * @param string $hook         'view_vars'
+ * @param string $type         'input/password'
+ * @param array  $return_value the current view vars
+ * @param array  $params       supplied params
+ *
+ * @return void|array
+ */
+function _elgg_disable_password_autocomplete($hook, $type, $return_value, $params) {
```

Comment:
I don't like doing browser's job. It could also be an accessibility issue. Will probably interfere with password generator apps.